sale_order_type_confirm_message/tests/test_sale_order_type_confirm_message.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Alberto Martínez <alberto.martinez@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class SomethingCase(TransactionCase):
```

Comment:
Maybe adding a test with wizard that confirms the sale order (improving coverage)